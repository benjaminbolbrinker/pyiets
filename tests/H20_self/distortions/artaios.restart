/home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode0_1 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/sp /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode2_1 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode1_1 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode1_0 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode2_0 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode0_0 