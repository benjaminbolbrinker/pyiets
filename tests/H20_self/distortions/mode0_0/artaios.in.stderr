"transmission.1.gpin", line 10: Can't plot with an empty x range!

